woocommerce/wc-admin
# Add Learn more about Product Settings note

Fixes #4201

Adds a "Learn more about Product Settings" admin note. It is added under these conditions:

- new site
- 24 hours after product add or import
- Not setting up for a client

### Screenshots

![image](https://user-images.githubusercontent.com/224531/83982924-c3385600-a96d-11ea-9054-b066b98860da.png)

### Detailed test instructions:

- Make sure you've gone through the onboarding wizard
- Make sure you have at least one product
- Find a product in the `wp_posts` table (any one will do) and set the `date_created` value back by at least 1 day
- Run the `wc_admin_daily` cron task
- The "Learn more about Product Settings" admin note should appear

### Changelog Note:

Enhancement: Add Learn more about Product Settings note
